modules/sympathy_answer.py

```diff
@@ -0,0 +1,14 @@
+import requests
+import json
+
+def aww(feel_for_ya):
+    print("How was your day today? Good? (\'y\'/'\'n\')"
```

Comment:
These messages will only get printed to STDOUT, which means they won't be sendt to the chat by the bot. The bot will only send a message to the chat when this function returns a String value to send, as the message is sent in the the main.py Bot object with the send_msg() function.

Also the parameter in aww(feel_for_ya) is unused, and the function later looks for a 'message' variable/parameter which doesn't exist.